Automattic/WP-Job-Manager
# Fix Job Dashboard table spacing and layout.

Fixes #1655 

#### Changes proposed in this Pull Request:

* Fix the original issue in #1655 of the Twenty Nineteen theme media query shrinking the table when it did not need to. 
  * _Note: in the original issue there is an "Applications" column._
* Change the inner cell to have min-width and the padding to be the same on all sides.
* Center the .date column text. 

#### Testing instructions:

* Install and activate the Twenty Nineteen theme.
* Load the Job Dashboard page.
* See if the table's spacing is consistent.
* Try other themes and repeat.

#### Proposed changelog entry for your changes:
Change spacing and padding inside #job-manager-job-dashboard div.
